doc/source/index.rst

```diff
@@ -1,13 +1,106 @@
-.. raw:: html
-
-    <form class="bd-search align-items-center" action="search.html" method="get">
-      <input type="search" class="form-control search-front-page" name="q" id="search-input" placeholder="&#128269; Search the docs ..." aria-label="Search the docs ..." autocomplete="off">
-    </form>
+:html_theme.sidebar_secondary.remove:
 
 .. _homepage:
 
 ArviZ: Exploratory analysis of Bayesian models
 ==============================================
+
+ArviZ is a Python package for exploratory analysis of Bayesian models. Includes functions for posterior analysis, data storage, sample diagnostics, model checking, and comparison.
+
+The goal is to provide backend-agnostic tools for diagnostics and visualizations of Bayesian inference in Python,
+by first converting inference data into `xarray <https://xarray.pydata.org/en/stable/>`_ objects.
+See :ref:`here <xarray_for_arviz>` for more on xarray and ArviZ usage
+and :ref:`here <schema>` for more on ``InferenceData`` structure
+and specification.
+
+.. raw:: html
```

Comment:
note: I like the wider layout with the images on the right over this one. But I think it is probably best to merge this for now, and have a follow up pr for the homepage and another for the examples/tags/descriptions. There are many things we should take into account to update the homepage (i.e. sponsors should be there! like we do on our [project page](https://www.arviz.org/en/latest/#sponsors-and-institutional-partners))